hook-consumer/src/consumer.rs

```diff
@@ -170,9 +185,21 @@ async fn spawn_webhook_job_processing_task<W: WebhookJob + 'static>(
         .await
         .expect("semaphore has been closed");
 
+    let labels = [
+        ("queue", webhook_job.queue()),
+        ("target", webhook_job.target()),
+    ];
+
+    metrics::increment_counter!("webhook_jobs_total", &labels);
+
     tokio::spawn(async move {
+        let now = tokio::time::Instant::now();
         let result = process_webhook_job(client, webhook_job, &retry_policy).await;
         drop(permit);
+
+        let elapsed = now.elapsed().as_secs_f64();
+        metrics::histogram!("webhook_jobs_processing_duration_seconds", elapsed, &labels);
```

Comment:
I wonder if this should only be called for non-timeouts? It seems like timeouts would pollute the metric since they would be truncated anyway, they could have their own metric (just a count)?

Reply:
Good point, we can have another counter for `webhook_jobs_timedout`.

Reply:
Alternatively, we could add an error label to the `webhook_jobs_failed` metric.

Reply:
This made me think that wouldn't tracking duration seconds on quick failures (like parsing errors) skew the metric lower? Kind of the opposite of tracking it for timeouts. I'm thinking of putting this histogram metric only on successful processing.